Jenkinsfile

```diff
@@ -19,9 +19,22 @@ pipeline {
     }
 
     stage('Run tests') {
-      steps {
-        sh './ci/test.sh'
-        junit 'tests/junit/*'
+      parallel {
+        stage("Test Ansible-Conjur-Collection") {
```

Comment:
Is this really just testing the Conjur (`conjur_variable`) lookup plugin? Could we make that more clear here?